datahub-web-react/src/app/entity/user/UserProfile.tsx

```diff
@@ -1,110 +1,411 @@
+            title: data?.corpUser?.info?.title || undefined,
+            team: data?.corpUser?.editableProperties?.teams?.join(',') || undefined,
+            email: data?.corpUser?.info?.email || undefined,
+            image: profileSrc,
+            slack: data?.corpUser?.editableProperties?.slack || undefined,
+            phone: data?.corpUser?.editableProperties?.phone || undefined,
+        };
     };
 
+    // About Text Change
+    const aboutMeTextChangeFunction = (inputString) => {
+        setEditableAboutMeText(inputString);
+        updateCorpUserPropertiesMutation({
+            variables: {
+                urn: data?.corpUser?.urn || '',
+                input: {
+                    aboutMe: inputString,
+                },
+            },
+        })
+            .catch((e) => {
+                message.destroy();
+                message.error({ content: `Failed to Save changes!: \n ${e.message || ''}`, duration: 3 });
+            })
+            .finally(() => {
+                message.success({
+                    content: `Changes saved.`,
+                    duration: 3,
+                });
+                refetch();
+            });
+    };
     return (
         <>
             {contentLoading && <Message type="loading" content="Loading..." style={messageStyle} />}
-            {data && data.corpUser && (
-                <LegacyEntityProfile
-                    title=""
-                    tags={null}
-                    header={getHeader(data.corpUser as CorpUser)}
-                    tabs={getTabs()}
+            <UserProfileWrapper>
+                <Row>
+                    <Col xl={5} lg={5} md={5} sm={24} xs={24}>
+                        <UserSidebar>
+                            <UserSidebarSubSection>
+                                <CustomAvatar
+                                    size={160}
+                                    photoUrl={profileSrc}
+                                    name={data?.corpUser?.info?.fullName || undefined}
+                                    style={{ marginTop: '14px' }}
+                                />
+                                <UserName>{data?.corpUser?.info?.fullName || <NotProvidedText />}</UserName>
+                                <UserRole>{data?.corpUser?.info?.title || <NotProvidedText />}</UserRole>
+                                <UserTeam>
+                                    {data?.corpUser?.editableProperties?.teams?.join(',') || <NotProvidedText />}
+                                </UserTeam>
+                                <Divider className="divider-infoSection" />
+                                <UserSocialDetails>
+                                    <Space>
+                                        <MailOutlined /> {data?.corpUser?.info?.email || <NotProvidedText />}
+                                    </Space>
+                                </UserSocialDetails>
+                                <UserSocialDetails>
```

Comment:
The amount of react code below this line is a lot - can we separate this into a separate sidebar component to make it easier to read?